includes/admin.form.inc

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * @file
+ * Admin form for this module.
+ */
+
+/**
+ * Admin form for presbydora.
+ *
+ * @param array $form
+ *   Form object.
+ * @param array $form_state
+ *   Form values.
+ *
+ * @return array
+ *   A renderable array available as a multi-select box.
+ */
+function presbydora_admin_form(array $form, array &$form_state) {
+  $tableselect = presbydora_build_tableselect();
+  $form['presbydora_access_bypass_pids'] = array(
+    '#type' => 'tableselect',
+    '#header' => $tableselect['header'],
+    '#options' => $tableselect['rows'],
+    '#multiple' => TRUE,
+    '#default_value' => variable_get('presbydora_access_bypass_pids', ''),
+  );
+
+  $form['presbydora_submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Save Configurations'),
```

Comment:
This shouldn't be pluralized. I suggest `Save configuration` to be consistant with the system_settings_form. https://api.drupal.org/api/drupal/modules%21system%21system.module/function/system_settings_form/7.x